src/main/resources/application.properties

```
@@ -1,9 +1,11 @@
-spring.datasource.url=jdbc:postgresql://kubernetes.docker.internal:5432/postgres
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://kubernetes.docker.internal:5432/postgis?useUnicode=yes&characterEncoding=UTF-8
+spring.datasource.username=postgis
+spring.datasource.password=postgis
 spring.datasource.driver-class-name=org.postgresql.Driver
 
 spring.jpa.hibernate.ddl-auto=none
-spring.datasource.initialization-mode=always
-spring.sql.init.schema-locations=classpath*:initDB/schema.sql
-spring.sql.init.data-locations=classpath*:initDB/data.sql
+spring.jpa.properties.hibernate.dialect = org.hibernate.spatial.dialect.postgis.PostgisDialect
+spring.sql.init.mode=always
+spring.sql.init.encoding= UTF-8
+spring.sql.init.schema-locations=classpath:/initDB/schema.sql
+spring.sql.init.data-locations=classpath:/initDB/data.sql
```

Опытным путем пришел к таким пропертям для инициализации. Пока больше ни разу не валились у меня. Предлагаю их использовать. 
```suggestion
spring.datasource.schema=classpath:initDB/schema.sql
spring.datasource.data=classpath:initDB/data.sql
```

---

Такие свойства помечаются Deprecated. Мб всё же не использовать?